cdk/lib/security-hq.ts

```diff
@@ -55,6 +55,10 @@ export class SecurityHQ extends GuStack {
         name: 'id',
         type: AttributeType.STRING,
       },
+      sortKey: {
+        name: 'dateNotificationSent',
+        type: AttributeType.STRING,
```

Comment:
Should this be `.NUMBER`?

Reply:
Nice one @jfsoul thanks for catching this 👍🏽